BUILD.gn

```diff
@@ -335,6 +342,58 @@ templated_file("electron_version_header") {
   args_files = get_target_outputs(":electron_version_args")
 }
 
+# This source set is just so we don't have to depend on all of //chrome/browser
+# You may have to add new files here during the upgrade if //chrome/browser/spellchecker
+# gets more files
+source_set("chrome_spellchecker") {
+  sources = [
+    "//chrome/browser/spellchecker/spell_check_host_chrome_impl.cc",
+    "//chrome/browser/spellchecker/spell_check_host_chrome_impl.h",
+    "//chrome/browser/spellchecker/spellcheck_custom_dictionary.cc",
+    "//chrome/browser/spellchecker/spellcheck_custom_dictionary.h",
+    "//chrome/browser/spellchecker/spellcheck_factory.cc",
+    "//chrome/browser/spellchecker/spellcheck_factory.h",
+    "//chrome/browser/spellchecker/spellcheck_hunspell_dictionary.cc",
+    "//chrome/browser/spellchecker/spellcheck_hunspell_dictionary.h",
+    "//chrome/browser/spellchecker/spellcheck_language_blacklist_policy_handler.cc",
+    "//chrome/browser/spellchecker/spellcheck_language_blacklist_policy_handler.h",
+    "//chrome/browser/spellchecker/spellcheck_language_policy_handler.cc",
+    "//chrome/browser/spellchecker/spellcheck_language_policy_handler.h",
+    "//chrome/browser/spellchecker/spellcheck_service.cc",
+    "//chrome/browser/spellchecker/spellcheck_service.h",
+  ]
+
+  if (has_spellcheck_panel) {
+    sources += [
+      "//chrome/browser/spellchecker/spell_check_panel_host_impl.cc",
+      "//chrome/browser/spellchecker/spell_check_panel_host_impl.h",
+    ]
+  }
+
+  if (use_browser_spellchecker) {
+    sources += [
+      "//chrome/browser/spellchecker/spelling_request.cc",
+      "//chrome/browser/spellchecker/spelling_request.h",
+    ]
+  }
+
+  public_configs = [
+    ":branding",
+    ":electron_lib_config",
```

Comment:
Why does this need `electron_lib_config` ?

Reply:
Include paths iirc

Reply:
Hmm, why would a chrome sources set require include paths for `//electron` ?

Reply:
@deepak1556 That patch to include the electron grit header